docs/testrunner-toolkit.md

```diff
@@ -45,48 +45,52 @@ Each docker image tag is the 'latest' image that supports the specific framework
 <!--START_AUTO_GENERATED_TABLE-->
```

Comment:
Hey @dpgraham -- Just curious about this note? It does not appear that anything between the `start ---end` comments is auto-generated? Are these comments leftover from an earlier version or do they mean something else?

Reply:
@sweeneyskirt-sl this just means that the table is generated elsewhere and copy-pasted here. I probably could have worded it better.